docs/reference/contributing/target/sai.md

```diff
@@ -0,0 +1,100 @@
+<h2 id="sai-port">Serial Audio Interface (SAI)</h2>
+
+The **Serial Audio Interface** allows you to send or receive an audio stream over a synchronous serial interface made of 3 to 4 lines.
+
+- SD: serial data.
+- BCLK: bit Clock.
+- WCLK: word Clock.
+- MCLK: main Clock (optional).
+
+A typical use case of this interface is with Codecs and DACs or ADCs to sample, process and output an audio signal.
+
+This highly configurable interface has a wide range of elements you can adjust:
+
+- Word length.
+- Data length (inside words).
+- Clocks polarity and phase.
+- Data alignment.
+
+Please refer to your device's reference manual for more details on its capabilities.
+
+This API provides a way to receive or send audio sample through this interface.
```

Comment:
I think lines 19 to 25 are not necessary here, it's supposed to be technology overview and/or some insight why someone should implement this API.

> Please refer to your device's reference manual for more details on its capabilities.

It's porting guide so that's obvious.

> This API provides a way to receive or send audio sample through this interface.
Already there in line 3.

> Transfer and free functions, plus an init function, comprise this API.
>+
>+`sai_init` is in charge of initializing this the whole interface and ensuring that the device supports the requested format. It also has to make sure the clocks are configured to generate the requested frequency with a reasonable error margin (depending on the master/slave tolerance).

Do not belong to this section, and it's part of specification.